turbo.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "http://turbo.build/schema.json",
+	"pipeline": {
+		"dev": {
+			"persistent": true,
+			"cache": false
+		},
+		"build": { "dependsOn": ["^build"] },
```

Comment:
This does something different then `^topological` we want the build to be topologically aware of itself so proper builds that need to happen happen first (like Wrangler), if we add `^topological` instead it would try to run them all at once.